content/english/covid-distance-checker-wip/_index.md

```diff
@@ -0,0 +1,101 @@
+A micro:bit is a programmable mini computer! It can be coded from any web browser in various languages like javaScript, Python, Scratch, and Blocks. For our session today with we will be using MakeCode blocks.
+
+Your micro:bit has the following physical features:
+
+- 25 individually-programmable LEDs
+- 2 programmable buttons
+- Physical connection pins
+- Motion sensors (accelerometer and compass)
+- Wireless communication via bluetooth radio
+- USB interface
+
+![alt-text](img/microbit-hardware-access.jpg)
+
+If you want to learn more about each individual component and all the features that come with your micro:bit, you can click [here](https://micro:bit.org/guide/features/).
+
+Go ahead and take out the the micro:bit and the rest of the components from the bag. You should have one micro:bit, 2 batteries, a micro USB cable, and the battery holder (the black box with the red and black wires coming out of it). If you're missing anything or need help feel free to ask one of the instructors!
+
+### Turning On the Radio
+Today we'll be using the Bluetooth radio on your micro:bit to figure out how far away you are from other micro:bits. The first thing we need to do is turn on the radio and tune it to a channel. Click on the section that says Radio. It's the pink section fifth from the top. Choose "radio set group 1" and drag it into the blue "on start" block in the workspace. Make sure that you use the same group number as everyone else. If you use different group numbers, your micro:bits won't be able to talk to each other.
+
+![MakeCode menu showing the Radio section selected](img/radioMenu.png)
+![MakeCode wokspace showing the "radio set group" block in the "on start" block](img/setRadioGroup.png)
+
+### Broadcasting a Signal
+Now that our micro:bits are set up to listen, we need to send out a signal for them to pick up. Click on the Radio section again. This time drag the "radio send number 0" block into the blue "forever" block in your workspace. This will make your micro:bit continuously send that number to any device listening in the same group.
+
+![MakeCode wokspace showing the "radio send number" block in the "forever" block](img/radioSendNumber.png)
+
+### Receiving a Message
+Now that we're sending messages and listening for them, we need to use that information to find out how close together or far apart our micro:bits are. To do that, we need to know how strong the signal we're receiving is. Go back into the Radio section and this time drag the "on radio received (receivedNumber)" block and drag it to an empty spot on your workspace. It should not be inside of any other block.
+
+This block tells our micro:bit to do something whenever it gets a number sent to it from another micro:bit.
+
+!["On radio received" block added to the workspace](img/onRadioReceived.png)
+
+Next, click on the red Variables section then click on "Make a Variable...". In the text box, type in "strength" as the name for your new variable, then click "Ok ✔".
```

Comment:
A discussion on variables would be helpful, since they might not know what a variable is.